dmoj/cptbox/landlock_helpers.h

```diff
@@ -0,0 +1,3 @@
+#include "landlock_header.h"
+
+int landlock_add_rules(const int ruleset_fd, const char **paths, __u64 access_rule);
```

Comment:
Could we name this `cptbox_landlock_add_rules`? Makes it easier to keep track of what's provided by the kernel versus by us.